docs/tutorial.rst

```diff
@@ -118,129 +118,118 @@ The Python interface
 Running a prexisting model
 --------------------------
 
-.. todo::
-    Add up to date code
 
-.. TODO port these old examples.
-
-.. ********
-.. Examples
-.. ********
-
-.. This content should go into a tutorial or somewhere else, but for now it's
-.. useful to keep a couple of short examples here for reference and
-.. to motivate the API.
-
-.. Models and genome data are split by species. Information about the genomes
-.. of a particular species is held in the ``genome`` class variable. So,
-.. suppose we wish to perform a simple simulation of human chromosome 22, we
-.. might have:
-
-.. .. code-block:: python
-
-..     import msprime
-..     from stdpopsim import homo_sapiens
-
-..     chrom = homo_sapiens.genome.chromosomes["chr22"]
-..     ts = msprime.simulate(
-..         sample_size=10,
-..         recombination_rate=chrom.default_recombination_rate,
-..         mutation_rate=chrom.default_mutation_rate,
-..         length=chrom.length)
-
-.. (This should be a very quick simulation, and the result will have very few
-.. variants, because although it performs a coalescent simulation of
-.. a 51,304,566bp chromosome, it does this with the effective population size of
-.. `Ne=1`.)
-
-.. The chromosome definitions also aware of recombination maps,
-.. which must be first downloaded. The default for ``homo_sapiens`` is ``HapmapII_GRCh37``,
-.. which we can find out, then download it as follows
-.. (the maps are stored in your ``~/.cache/stdpopsim/`` directory):
-
-.. .. code-block:: python
+.. _sec_tutorial_generic_models:
 
-..    homo_sapiens.genome.default_genetic_map
-..    # 'HapmapII_GRCh37'
-..    gmap = homo_sapiens.HapmapII_GRCh37()
-..    gmap.download()
+**************
+Generic models
+**************
 
+In this example, we will use the ``stdpopsim`` API to simulate a generic
+model for a particular species. We will use the information for a particular
+species and instantiate the model directly.
 
-.. After this has been done (once only), we can run simulations using this genetic map as follows:
+Here, we will simulate 10% of human chromosome 22 under a constant size
+population model, using the current best estimate of the human
+effective population size from the :ref:`sec_catalog`.
 
-.. .. code-block:: python
 
-..     chrom = homo_sapiens.genome.chromosomes["chr22"]
-..     ts = msprime.simulate(
-..         sample_size=10,
-..         mutation_rate=chrom.default_mutation_rate,
-..         recombination_map=chrom.recombination_map())
+1. Import the necessary packages
 
-.. Recombination maps will be downloaded on demand and cached in a
-.. platform-appropriate user cache directory (e.g., ``$HOME/.cache/stdpopsim`` on
-.. Linux). In this example we didn't specify which recombination map we want, and so the
-.. API will use the default. We can also ask for specific maps, if we want:
+.. code-block:: python
 
-.. .. code-block:: python
+    >>> import msprime
```

Comment:
We don't actually need msprime here, do we?

Reply:
You are right. Good catch.